docs/data-sources/gateway.md

```diff
@@ -0,0 +1,236 @@
+
+Read-Only:
+
+- `arn` (String)
+- `execution_role` (Attributes) (see [below for nested schema](#nestedatt--aws_fargate_task_definition--execution_role))
+- `service` (Attributes) (see [below for nested schema](#nestedatt--aws_fargate_task_definition--service))
+- `task_role` (Attributes) (see [below for nested schema](#nestedatt--aws_fargate_task_definition--task_role))
+
+<a id="nestedatt--aws_fargate_task_definition--execution_role"></a>
+### Nested Schema for `aws_fargate_task_definition.execution_role`
+
+Read-Only:
+
+- `arn` (String)
+
+
+<a id="nestedatt--aws_fargate_task_definition--service"></a>
+### Nested Schema for `aws_fargate_task_definition.service`
+
+Read-Only:
+
+- `arn` (String)
+- `cluster` (Attributes) (see [below for nested schema](#nestedatt--aws_fargate_task_definition--service--cluster))
+- `security_groups` (List of Object) (see [below for nested schema](#nestedatt--aws_fargate_task_definition--service--security_groups))
+- `subnets` (List of Object) (see [below for nested schema](#nestedatt--aws_fargate_task_definition--service--subnets))
+
+<a id="nestedatt--aws_fargate_task_definition--service--cluster"></a>
+### Nested Schema for `aws_fargate_task_definition.service.cluster`
+
+Read-Only:
+
+- `arn` (String)
+
+
+<a id="nestedatt--aws_fargate_task_definition--service--security_groups"></a>
+### Nested Schema for `aws_fargate_task_definition.service.security_groups`
+
+Read-Only:
+
+- `id` (String)
+
+
+<a id="nestedatt--aws_fargate_task_definition--service--subnets"></a>
+### Nested Schema for `aws_fargate_task_definition.service.subnets`
+
+Read-Only:
+
+- `arn` (String)
+- `az` (String)
+- `vpc` (Object) (see [below for nested schema](#nestedobjatt--aws_fargate_task_definition--service--subnets--vpc))
+
+<a id="nestedobjatt--aws_fargate_task_definition--service--subnets--vpc"></a>
+### Nested Schema for `aws_fargate_task_definition.service.subnets.vpc`
+
+Read-Only:
+
+- `id` (String)
+
+
+
+
+<a id="nestedatt--aws_fargate_task_definition--task_role"></a>
+### Nested Schema for `aws_fargate_task_definition.task_role`
+
+Read-Only:
+
+- `arn` (String)
+
+
+
+<a id="nestedatt--cluster_ip"></a>
+### Nested Schema for `cluster_ip`
+
+Read-Only:
+
+- `name` (String)
+
+
+<a id="nestedatt--gcp_cloud_run"></a>
+### Nested Schema for `gcp_cloud_run`
+
+Read-Only:
+
+- `id` (String)
+- `serverless_vpc_connector` (Attributes) (see [below for nested schema](#nestedatt--gcp_cloud_run--serverless_vpc_connector))
+- `service_account` (Attributes) (see [below for nested schema](#nestedatt--gcp_cloud_run--service_account))
+
+<a id="nestedatt--gcp_cloud_run--serverless_vpc_connector"></a>
+### Nested Schema for `gcp_cloud_run.serverless_vpc_connector`
+
+Read-Only:
+
+- `id` (String)
+
+
+<a id="nestedatt--gcp_cloud_run--service_account"></a>
+### Nested Schema for `gcp_cloud_run.service_account`
+
+Read-Only:
+
+- `self_link` (String)
+
+
+
+<a id="nestedatt--k8s_container"></a>
+### Nested Schema for `k8s_container`
```

Comment:
It's not really a container, right? But a deployment?

Reply:
well this is actually the container, with a reference to the deployment it's in. We could conflate the two if we want.